lib/view/airway/checklist/intubation_checklist_infographic_page.dart

```diff
@@ -1,11 +1,17 @@
 import 'package:flutter/material.dart';
 
+import '../../../constants.dart';
 import '../../../style.dart';
+import '../../../utils/firebase.dart';
 import '../../reusable_infographic_page.dart';
 
 class IntubationChecklistInfographicPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    // Analytics set screen name, stays until another screen changes it
```

Comment:
I'm not very familiar with Google analytics client, but if its already tracking route navigation events, what does this add as there are no other analytics events being tracked on this screen?

Reply:
See above ☝️